volontulo/tests/views/test_organizations.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+
+u"""
+.. module:: test_organizations
+"""
+from django.test import Client
+from django.test import TestCase
+
+from volontulo.models import Organization
+from volontulo.tests.views.test_usersprofile import TestUsersProfile
+
+
+class TestOrganizations(TestCase):
+    u"""Class responsible for testing organization specific views."""
+
+    @classmethod
+    def setUpTestData(cls):
+        u"""Set up data for all tests."""
+        # volunteer user - totally useless
+        TestUsersProfile.initialize_empty_volunteer()
+        # organization user - no offers
+        TestUsersProfile.initialize_empty_organization()
+
+    def setUp(self):
+        u"""Set up each test."""
+        self.client = Client()
+
+    # pylint: disable=invalid-name
+    def test__create_organization_get_form_anonymous_user(self):
+        u"""Test getting form for creating organization as anonymous"""
+        # Disable for anonymous user
+        response = self.client.get('/organizations/create')
+
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(
+            response,
+            'http://testserver/login?next=/organizations/create',
+            302,
+            200,
+        )
+
+    # pylint: disable=invalid-name
+    def test__create_organization_get_form_authorized_user(self):
+        u"""Test getting form for creating organization as authorized"""
+        self.client.post('/login', {
+            'email': u'volunteer1@example.com',
+            'password': 'volunteer1',
+        })
+        response = self.client.get('/organizations/create')
+
+        self.assertTemplateUsed(
+            response,
+            'organizations/organization_form.html'
+        )
+        # pylint: disable=no-member
+        self.assertIn('organization', response.context)
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, u'Tworzenie organizacji')
+
+    # pylint: disable=invalid-name
+    def test__create_organization_post_form_anonymous_user(self):
```

Comment:
If anonymous user want to enter this site there should be redirection to login page. If user is authorized then creating organization form is returned.